core/src/main/java/hudson/FilePath.java

```diff
@@ -2266,12 +2267,15 @@ private Integer writeToTar(File baseDir, DirScanner scanner, OutputStream out) t
 
     /**
      * Reads from a tar stream and stores obtained files to the base dir.
+     * @since TODO: the method uses 
```

Comment:
Any needed information can be mentioned in javadoc description, you can ignore it and continue wasting your time in git history.
since - when first appeared. In description <p> since 1.XXX.2 - supports large file > 10 GB